openfeature-provider/js/src/version.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Version information for the Confidence OpenFeature local provider.
+ * This version is updated automatically by release-please.
+ */
+export const VERSION = '0.1.1'; // x-release-please-version
```

Comment:
Could it be worth adding a check in CI that this version and the one in `package.json` are aligned?